Owl/gradlew.bat

```diff
@@ -64,29 +64,14 @@ echo location of your Java installation.
 
 goto fail
 
-:init
-@rem Get command-line arguments, handling Windows variants

```

Comment:
Don't know what happened here but I guess all the gradle changes don't affect anything

Reply:
If you update gradle "properly" i.e. `./gradlew wrapper --gradle-version 6.7` rather than just changing the version in .properties then it also updates the wrapper i.e. these changes.